content/chapter 3/3.3-sync.md

```diff
@@ -340,7 +343,15 @@ func worker(name string, wg *sync.WaitGroup) {
 
 فرض کنید می‌خواهیم چند فایل را به صورت موازی دانلود کنیم و منتظر بمانیم تا همه دانلودها کامل شوند:
 
-```go
+{{< play >}}
```

Comment:
Error: error building site: process: readAndProcessContent: "/home/runner/work/book/book/content/chapter 3/3.3-sync.md:346:1": failed to extract shortcode: shortcode "play" must be closed or self-closed

https://github.com/GoFarsi/book/actions/runs/17193583581/job/48772606095#step:4:9